docs/scripting/troubleshooting.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Troubleshooting JavaScript
+category: Documentation
+---
+
+This article teaches you the basic workflow for debugging any JavaScript issue.
+
+## Error Reference
+
+These errors can be a helpful debugging your code, but the reported problem isn't always immediately clear. Most of the errors are common in most JavaScript engines, so most of the following errors can be found with a search on a search engine (e.g. Google Search).
```

Comment:
Most of these errors are common in JavaScript engines, so they can be found with a search on a search engine (e.g. Google Search).  To avoid repetition of the word most.